pkg/admissioncontroller/apis/config/v1alpha1/types.go

```diff
@@ -31,6 +31,8 @@ type AdmissionControllerConfiguration struct {
 	GardenClientConnection componentbaseconfigv1alpha1.ClientConnectionConfiguration `json:"gardenClientConnection"`
 	// LogLevel is the level/severity for the logs. Must be one of [info,debug,error].
 	LogLevel string `json:"logLevel"`
+	// LogFormat is the format for the logs. Must be one of [json,text].
```

Comment:
```suggestion
	// LogFormat is the format for the logs. Must be one of [json,text].
	// Defaults to "json".
```

You could also update similarly the internal types.go.